infra/app/service/main.tf

```diff
@@ -22,25 +22,16 @@ data "aws_subnets" "private" {
 }
 
 locals {
-  # The prefix key/value pair is used for Terraform Workspaces, which is useful for projects with multiple infrastructure developers.
-  # By default, Terraform creates a workspace named “default.” If a non-default workspace is not created this prefix will equal “default”, 
-  # if you choose not to use workspaces set this value to "dev" 
-  prefix = terraform.workspace == "default" ? "" : "${terraform.workspace}-"
```

Comment:
in the process of adding file upload job configs, I consolidated all the service-related configs into a service_config object in env-config module's output, so we ended up moving the name logic into the config module, cleaning up this root module